codex-rs/tui/src/slash_command.rs

```diff
@@ -16,6 +16,7 @@ pub enum SlashCommand {
     Diff,
     Quit,
     ToggleMouseMode,
+    Compact,
```

Comment:
As noted on line 13, enum order should be what we think frequency order is. I would list this second (after `New`).